src/markdownAdapters.ts

```diff
@@ -66,6 +66,26 @@ function resolveAllOf(
     )
 }
 
+function resolveOneOf(
+    schemaObject: OpenAPIV3.SchemaObject,
+    refs: IRefs,
+    depth: number | undefined = undefined,
+    depthCounter: number = 0,
+): OpenAPIV3.SchemaObject {
+    if (!schemaObject.oneOf) {
+        return schemaObject
+    }
+
+    schemaObject.oneOf = schemaObject.oneOf.map(
```

Comment:
Do we *need* to modify `schemaObject` in place? Maybe we could `return {...schemObject, oneOf: schemaObject.oneOf.map(...)}`?